README.md

```diff
@@ -121,6 +121,10 @@ export interface JSONVisitor {
      * Invoked when a comma or colon separator is encountered. The offset and length represent the location of the separator.
      */
     onSeparator?: (charcter: string, offset: number, length: number) => void;
+    /**
+     * Invoked when a line break is encountered. The offset and length represent the location of the line break.
+     */
+    onLineBreak?: (lineNumber: number, offset: number, length: number) => void;
```

Comment:
`lineNumber` ? Isn't it only offset / length

Reply:
Good catch - had line number previously, forgot to update the readme.